src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -199,6 +199,12 @@ void handleDirectCommand(Channel channel, User sender, String payload) {
             return;
         }
 
+        m = Pattern.compile("archive (?:github )repo (\\S+) and prefix repo description with (\\S+)",CASE_INSENSITIVE).matcher(payload);
```

Comment:
I would suggest simplifying this dropping all the words and taking the first string after repo (it can't have a space) and then the next is the description

Reply:
I've dropped everything. I though I could put word(s) between the capture groups, like the "to" in the `rename` command, it will do without.